logstash-core/lib/logstash/queue_factory.rb

```diff
@@ -32,7 +28,7 @@ def self.create(settings)
           settings.get("pipeline.batch.size") * settings.get("pipeline.workers") * 2
         )
       else
-        raise ConfigurationError, "Invalid setting `#{queue_type}` for `queue.type`, supported types are: 'memory_acked', 'memory', 'persisted'"
+        raise ConfigurationError, "Invalid setting `#{queue_type}` for `queue.type`, supported types are: 'memory' or 'persisted'"
```

Comment:
Should we register this message with i18n while we're at it?

Reply:
@yaauie not right now while this is in flux imo (especially since i18n == Ruby :))